### ISSUE TITLE
Remove the add food page and show it on the home page.

### ISSUE THREAD

**Issue by mohit200008** [mohit200008/FoodSaver20008#6]
Add a plus icon on the home page and When a user clicks on that plus icon The form pops up asking for details

**Comment by shefali12-ab** [comment]
@mohit200008 Sir, Can I work on this issue?

**Comment by mohit200008** [comment]
sure @shefali12-ab

**Comment by mohit200008** [comment]
@shefali12-ab  any update on the issue ????

**Comment by shefali12-ab** [comment]
@mohit200008 yes sir I am working on it, little busy these days. will update you soon.

**Comment by mohit200008** [comment]
@shefali12-ab any update on the issue ????

**Comment by shefali12-ab** [comment]
@mohit200008 yes sir I am working on it.

**Comment by aktamoli0910** [comment]
Any update @shefali12-ab ??

**Comment by Tushar98644** [comment]
can i work on this? @mohit200008

**Comment by mohit200008** [comment]
It's already done but what you can do is display the information user enters on a seperate page

**Comment by Tushar98644** [comment]
so should i make a new display page for the ingredients?

**Comment by Tushar98644** [comment]
> It's already done but what you can do is display the information user enters on a seperate page

it;s not yet done i think...the add food page is still there and there is no plus icon on the home page

**Comment by mohit200008** [comment]
Okay so can you do both?

**Comment by Tushar98644** [comment]
ya i can do both the parts.....but can i create a new issue for the display page?

**Comment by mohit200008** [comment]
Okay you can do both
